FinniversKit/Sources/Recycling/GridViews/AdRecommendations/GridView/AdRecommendationsGridView.swift

```diff
@@ -234,7 +234,7 @@ extension AdRecommendationsGridView: AdRecommendationsGridViewLayoutDelegate {
         case .columns(let columns) where columns > 1 && columns <= 3:
             return columns
         default:
-            if traitCollection.preferredContentSizeCategory.isAccessibilityCategory {
+            if traitCollection.preferredContentSizeCategory.isAccessibilityCategory && Config.isDynamicTypeEnabled {
```

Comment:
Hmm, what do you think about a rewrite like this?
```
default:
   var columns = traitCollection.horizontalSizeClass == .regular ? 3 : 2
   if traitCollection.preferredContentSizeCategory.isAccessibilityCategory && Config.isDynamicTypeEnabled {
      columns -= 1
   }
   return columns
```

Reply:
Looks smooth 👍